Xamarin.Forms.Platform.iOS/CollectionView/HeightConstrainedTemplatedCell.cs

```diff
@@ -12,8 +12,9 @@ public HeightConstrainedTemplatedCell(CGRect frame) : base(frame)
 
 		public override void ConstrainTo(CGSize constraint)
 		{
+			base.ConstrainTo(constraint);
+
 			ConstrainedDimension = constraint.Height;
-			Layout(constraint);
```

Comment:
@hartez Previously, there was a `Layout()` call inside `ConstrainTo()`, but `PreferredLayoutAttributesFittingAttributes` does layout too. Any reason for that? Removing the call here made cell sizing better.

Reply:
TBH, I don't know why there was a Layout call there. Seems to work fine without it.